src/utils/db.ts

```diff
@@ -0,0 +1,48 @@
+import { openDB, DBSchema, IDBPDatabase } from 'idb';
+
+import { Todo } from '../redux/todosSlice';
+
+interface MyDb extends DBSchema {
+  todosStore: {
+    key: string;
+    value: Todo[];
+  };
+  appStore: {
```

Comment:
I don't think adding `Store` to the end is a good idea. There is not "store" context if we are talking about a database.

Just leave `todos, app`